public/js/components/app.js

```diff
@@ -70,29 +71,38 @@ export class App extends Component {
       selectedTileLayer: null,
       selectedFileLayer: null,
       selectedLanguage: 'default',
+      selectedColor: null,
+      selectedColorOp: 'overlay',
+      selectedPercentage: 0.25,
```

Comment:
nit, I could be wrong, but I don't think we need to set defaults for these.

```suggestion
      selectedColorOp: null,
      selectedPercentage: null,
```